stack-9.2.1.yaml

```diff
@@ -0,0 +1,2 @@
+resolver: nightly-2021-11-06
+compiler: ghc-9.2.1
```

Comment:
nice trick, I didn't realize we didn't have to wait until stackage officially switched nightlies to 9.2 before we can use that version of the compiler with stack!